### ISSUE TITLE
Addition of new head circumference charts

### ISSUE THREAD

**Issue by soorajpatil** [p2/growth-charts#18]
Hi.. This is Dr Sooraj, i am pediatrician.. The app is very good.. Saves lots of time.. Kindly add more head circumference charts from who and cdc ,atleast ad head circumference who o to 5 years, charts for 1 to 18 years from CDC.. They help in bigger children

**Comment by soorajpatil** [comment]
![image](https://cloud.githubusercontent.com/assets/9763365/5057138/3e9f7dce-6cd3-11e4-85b9-cc2b4b0d52a7.jpg)

**Comment by usnish** [comment]
Yes please! We're working in the field and don't really have any alternatives, so an additional chart would be super helpful!
